test/test_conda_mirror.py

```diff
@@ -93,30 +97,11 @@ def test_cli(tmpdir, channel, platform, repodata):
         assert len(rd['info']) == len(disk_info)
         assert len(rd['packages']) == len(disk_packages)
 
-
-def test_handling_bad_package(tmpdir, repodata):
```

Comment:
Do the assertions in the test that calls this new function cover the purpose of the assertions in the removed `test_handling_bad_package`?